crates/gpui/src/platform/blade/blade_renderer.rs

```diff
@@ -331,6 +333,16 @@ impl BladeRenderer {
         window: &I,
         config: BladeSurfaceConfig,
     ) -> anyhow::Result<Self> {
+        let sample_count = std::env::var("ZED_SAMPLE_COUNT")
```

Comment:
This environment variable was called `ZED_PATH_SAMPLE_COUNT` before. We may need to update the documentation in `docs/src/linux.md` after this change. :slightly_smiling_face: